pom.xml

```diff
@@ -0,0 +1,105 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>2.26</version>
+  </parent>
+
+  <!--TODO: this artifact ID may be unclear-->
+  <artifactId>agent-remoting-support</artifactId>
```

Comment:
I would say "jnlp" has nothing to do with Remoting protocols. We just call them `JNLPAgentProtocol`s due to historical reasons